src/unshield.c

```diff
@@ -12,6 +12,7 @@
 #include <stdlib.h>
 #include <string.h>
 #include <unistd.h>
+#include <getopt.h> /* getopt at: https://gist.github.com/ashelly/7776712 */
```

Comment:
In Linux, getopt(), which unshield uses, is part of unistd.h but getopt_long(), that is not used by unshield, is in getopt.h. A bit confusing maybe, but I believe that the current code is correct as-is. See http://man7.org/linux/man-pages/man3/getopt.3.html

Reply:
Ok, I just checked this on FreeBSD and MacOSX and it's the same there as well. And I get it now that `win32_msvc/unistd.h` should mimic this behavior. Then it appears that the `#pragma once` in `win32_msvc/getopt.h` doesn't work properly in my VS2017 build.

Reply:
It seems that `#pragma once` doesn't work as expected because it's used in two different compile units (`libunshield` and `unshield`). Why the symbols get into `libunshield` is a mystery to me, as `getopt()` isn't used in `libunshield` and I would expect it to get dropped by the linker.